go/consensus/tendermint/tendermint.go

```diff
@@ -1615,6 +1665,13 @@ func init() {
 	Flags.Bool(CfgDebugDisableCheckTx, false, "do not perform CheckTx on incoming transactions (UNSAFE)")
 	Flags.Bool(CfgDebugUnsafeReplayRecoverCorruptedWAL, false, "Enable automatic recovery from corrupted WAL during replay (UNSAFE).")
 
+	// State sync.
+	Flags.Bool(CfgConsensusStateSyncEnabled, false, "enable state sync")
+	Flags.StringSlice(CfgConsensusStateSyncConsensusNode, []string{}, "consensus node to use for syncing the light client")
+	Flags.Duration(CfgConsensusStateSyncTrustPeriod, 24*time.Hour, "light client trust period")
+	Flags.Uint64(CfgConsensusStateSyncTrustHeight, 0, "light client trusted height")
+	Flags.String(CfgConsensusStateSyncTrustHash, "", "light client trusted consensus header hash")
```

Comment:
Update these comments to refer specifically to state sync as there will later be a similar set of options for the light node.